docsrc/dynamo/torch_compile.rst

```diff
@@ -38,7 +38,7 @@ Custom Setting Usage
     ...
     optimized_model = torch.compile(model, backend="torch_tensorrt", dynamic=False,
                                     options={"truncate_long_and_double": True,
-                                             "precision": torch.half,
+                                             "enabled_precisions": [torch.half],
```

Comment:
Can you use set notation here since that is the class we normally recommend? e.g. `{torch.float, torch.half}`